operator/internal/validation/openshift/common.go

```diff
@@ -1,15 +1,21 @@
 package openshift
 
 import (
+	"regexp"
 	"strings"
 
 	lokiv1beta1 "github.com/grafana/loki/operator/apis/loki/v1beta1"
 
-	"github.com/grafana/loki/pkg/logql/syntax"
 	"github.com/prometheus/prometheus/model/labels"
+
+	"github.com/grafana/loki/pkg/logql/syntax"
 )
 
 const (
+	severityLabelName    = "severity"
+	summaryLabelname     = "summary"
```

Comment:
summaryLabelName

Reply:
Good catch `summary` and `description` need to be suffixed with `AnnotationName` instead of `LabelName`.